src/device_registry.c

```diff
@@ -21,6 +22,7 @@ void init_devices()
     g533_init(&devicelist[2]);
     g633_init(&devicelist[3]);
     g930_init(&devicelist[4]);
+    g933_init(&devicelist[6]);
     arctis_init(&devicelist[5]);
```

Comment:
Also you need to swap the numbers then, because when someone else adds another device, he can easily miscount it.